src/ui/components/Library/Team/View/NewTestRuns/Overview/TestRunOverviewContent.tsx

```diff
@@ -29,27 +30,27 @@ export function TestRunOverviewContent() {
   const testRun = testRuns.find(testRun => testRun.id === testRunId);
 
   let children = null;
-  if (testRun && recordings) {
-    if (!hasFilters || testRuns.find(testRun => testRun.id === testRunId)) {
-      children = (
-        <>
-          <RunSummary
-            isPending={isPending}
-            testRun={testRun}
-            durationMs={durationMs}
-            setTestFilterByText={setFilterTestsByText}
-            testFilterByText={filterTestsByText}
-            setFilterCurrentRunByStatus={setFilterCurrentRunByStatus}
-            filterCurrentRunByStatus={filterCurrentRunByStatus}
-          />
-          <RunResults
-            isPending={isPending}
-            testFilterByText={filterTestsByText}
-            filterCurrentRunByStatus={filterCurrentRunByStatus}
-          />
-        </>
-      );
-    }
+  if (testRun && recordings && !hasFilters) {
```

Comment:
I think `!hasFilters` should be removed? or maybe moved into its own branch where we show a different message if there's no selection _and_ there are filters applied.

Reply:
Agreed. Let's tackle in scs-1709